ew/utils/loop.py

```diff
@@ -858,6 +859,87 @@ async def enemy_action_tick_loop(id_server):
+                            slimeloss = min(50,slime)
+                            injury_text = ["{name} tries to stay balanced but fails, falling to the ground, catching a few grazes on the arm they used to brace. **They lose {slime} slime!**".format(name=player.display_name, slime=slimeloss),"{name} Walks into a lamppost. **They lose {slime} slime!**".format(name=player.display_name, slime=slimeloss)]
+                                
+                        
+                    elif inebriation<=severityLimit.get("high"):
+                        if savingthrow > 12:
+                            injury_text = ["{name} begins to tilt over, before they can fall and faceplant the pavement they catch themselves on a railing. Nice save!".format(name=player.display_name),"{name} gurks.".format(name=player.display_name)]
+                        else:
+                            slimeloss = min(250,slime)
+                            injury_text = ["{name} suddenly faceplants onto the floor from standing leaving a nasty bruise. **They lose {slime} slime!**".format(name=player.display_name, slime = slimeloss)]
+                    
+                    
+                    elif inebriation<=severityLimit.get("wasted"):
+                        if savingthrow > 15:
+                            injury_text = ["{name} attempts to walk, stumbles slightly but manages to correct themselves. Good job!".format(name=player.display_name,slime=slimeloss),"{name} eructates.".format(name=player.display_name)]
+                        else:
+                            slimeloss = slime * 0.05 # 5% slimeloss
+                            injury_text = ["{name} attempts to walk, accidentally standing on the side of their foot. Their ankle is almost instantly sprained and in their pained hopping they manage to topple over a nearby railing falling backfirst onto the pavement. **They lose {slime} slime!**".format(name=player.display_name,slime=slimeloss)]
+                    
+                    
+                    elif inebriation<=severityLimit.get("wrecked"):
+                        if savingthrow > 18:
+                            injury_text = ["{name} drunkenly punches a nearby window but the glass doesn't break. Phew!".format(name=player.display_name,slime=slimeloss)]
+                        else:
+                            slimeloss = slime * 0.1 # 10% slimeloss
+                            injury_text = ["{name} drunkenly punches a nearby window, the glass shatters and their hand is cut to ribbons by the resultant shards. **They lose {slime} slime!**".format(name=player.display_name,slime=slimeloss)]
+                    
+                    
+                    else:
+                        if savingthrow == 20:
+                            injury_text = ["{name} mindlessly stumbles out into a nearby road. An oncoming car bleats and {name} hears it! Only just scrambling away from the oncoming car. Nice save!".format(name=player.display_name)]
```

Comment:
"Only just scrambling away from the oncoming car" don't quite work. Probably change to "They only just scramble away from the oncoming car."